src/BetterAppListBuilder.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+/**
+ * @file
+ * Apigee Edge UI module for Drupal.
+ *
+ * Copyright (C) 2019 PRONOVIX GROUP BVBA.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301,
+ * USA.
+ */
+
+namespace Drupal\apigee_edge_ui;
+
+use Drupal\apigee_edge\Entity\AppInterface;
+use Drupal\apigee_edge\Entity\ListBuilder\DeveloperAppListBuilderForDeveloper;
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\StringTranslation\TranslatableMarkup;
+use Drupal\Core\Template\Attribute;
+
+
+/**
+ * Advanced list builder for developer apps.
+ */
+class BetterAppListBuilder extends DeveloperAppListBuilderForDeveloper {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function render() {
+    $build['app_list'] = [
```

Comment:
Would not it be possible to call `$build = parent::render()` and alter the returned build array instead of re-inventing the wheel here?